pom.xml

```diff
@@ -72,28 +69,39 @@
 			<artifactId>aws-cloudwatch-library</artifactId>
 			<version>1.11.136</version>
 			<optional>true</optional>
+			<exclusions>
+				<!-- Provided by apache-httpcomponents-client-4-api plugin -->
+				<exclusion>
+					<groupId>org.apache.httpcomponents</groupId>
+					<artifactId>httpclient</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.httpcomponents</groupId>
+					<artifactId>httpcore</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>apache-httpcomponents-client-4-api</artifactId>
+			<version>4.5.14-150.v7a_b_9d17134a_5</version>
+			<optional>true</optional>
+		</dependency>
```

Comment:
I suggest to close that PR and to create a new PR with just that

Reply:
I see no reason to follow your suggestion.

Reply:
since adding the bouncycastle-api dependency seems not correct to me

Reply:
You have not provided any reasoning as to why it is not correct. I have provided reasoning as to why it _is_ correct here:

https://github.com/jenkinsci/monitoring-plugin/pull/11#discussion_r1275392505

Reply:
=> commit e299c74fdef19af0a0cc3f334fffd8a85349c004